lib/Handler/FileHandler.php

```diff
@@ -21,13 +21,10 @@ public function __construct($minimalLevel, $verbose, $additionalParameters = arr
         }
         $filename = $additionalParameters['filename'];
         parent::__construct($minimalLevel, $verbose);
-        if ((!file_exists($filename) && file_put_contents($filename, '') === false) ||!is_writable($filename)) {
+        if ((!file_exists($filename) && file_put_contents($filename, '') === false) || !is_writable($filename)) {
```

Comment:
Dangerous as `file_put_contents($filename, '')` would immediately overwrite the file.